Survey/Sources/Presentation/Modules/Home/HomeViewModel.swift

```diff
@@ -44,14 +45,16 @@ extension HomeViewModel: ViewModel {
                     .asDriver()
             }
             .switchToLatest()
-            .map {
+            .map { (cachedStorageUseCase.load(), $0) }
+            .map { args in
+                let (cachedSurveys, resultSurveys) = args as? ([APISurvey], [APISurvey]) ?? ([], [])
                 pageNumber += 1
-                if !UserStorage.cachedSurveyList.hasSameChildren(as: $0 as? [APISurvey] ?? []) {
-                    UserStorage.cachedSurveyList = output.surveys + $0 as? [APISurvey] ?? []
-                    return output.surveys + $0
+                if !cachedSurveys.hasSameChildren(as: resultSurveys) {
+                    cachedStorageUseCase.store(data: output.surveys + resultSurveys as? [APISurvey] ?? [])
+                    return output.surveys + resultSurveys
```

Comment:
```suggestion
                    let updatedSurveys = output.surveys + resultSurveys as? [APISurvey] ?? []
                    cachedStorageUseCase.store(data: updatedSurveys)
                    return updatedSurveys
```

Reply:
Thank you :D, I fixed it at https://github.com/nkhanh44/ic-survey-ios/pull/85/commits/b5dbcfd678fbc51d3270ea0f32badd3b1eaea60d.